packages/cli/commands/worker.ts

```diff
@@ -71,7 +79,7 @@ export class Worker extends Command {
 	 * get removed.
 	 */
 	static async stopProcess() {
-		console.log(`\nStopping n8n...`);
+		LoggerProxy.getInstance().info(`\nStopping n8n...`);
```

Comment:
Same as prior comment.

```suggestion
		LoggerProxy.info(`\nStopping n8n...`);
```